README.md

```diff
@@ -1,15 +1,21 @@
-# stolostron
-Getting started with Red Hat Advanced Cluster Management for Kubernetes?  This repo contains tools, integrations, and suggestions to jumpstart your multicluster and multicloud presence!
+# Welcome to the Stolostron Project
+Getting started with Red Hat Advanced Cluster Management for Kubernetes?  You're in the right place!  The Stolostron project houses the projects built into [Red Hat Advanced Cluster Management for Kubernetes](https://www.redhat.com/en/technologies/management/advanced-cluster-management).  This repo contains tools, integrations, and suggestions to jumpstart your multicluster and multicloud presence!
```

Comment:
Just a thought but wonder if there's benefit to adding a few words in here that indicate RHACM works on * k8s. Heard recently customers default to thinking RHACM is only for Openshift, and we could sneak in a few words here to fix that incorrect perception!

Reply:
@nelsonjean good suggestion!  Some of RHACM's hub only works on OpenShift - buuuut we can manage *KS so let me add a blerb!